keylime/cloud_verifier_tornado.py

```diff
@@ -855,6 +866,66 @@ async def invoke_provide_v(agent):
         asyncio.ensure_future(process_agent(agent, states.GET_QUOTE))
 
 
+async def invoke_notify_error(agent, tosend):
+    if agent is None:
+        logger.warning("Agent deleted while being processed")
+        return
+    kwargs = {
+        "data": tosend,
+    }
+    if agent["ssl_context"]:
```

Comment:
This works only if all the agents event loops are running on a single process. If not you will need to recreate the contexts using `web_util.generate_agent_mtls_context(...)`.

Reply:
Thanks for pointing that out. I've removed the caching but added context recreation for connecting to other agents.